src/test_lib_json/CMakeLists.txt

```diff
@@ -1,39 +1,29 @@
 # vim: et ts=4 sts=4 sw=4 tw=0
 
-add_executable(jsoncpp_test
-    jsontest.cpp
-    jsontest.h
-    fuzz.cpp
-    fuzz.h
-    main.cpp
-)
+ADD_EXECUTABLE(jsoncpp_test
```

Comment:
Why is this change needed?  case change and whitespace change seems unnecessary.

Reply:
This is usually caused by automatic CLion reformatting.

Reply:
Any action you would like me to take sir, I have already done some formatting of that section of code, let me know if it is to your approval.